.yamato/com.unity.ml-agents-test.yml

```diff
@@ -1,21 +1,16 @@
 test_editors:
   - version: 2018.4
     # 2018.4 doesn't support code-coverage
-    coverageOptions:
-    minCoveragePct: 0
+    enableCodeCoverage: !!bool false
```

Comment:
See caveat in https://internaldocs.hq.unity3d.com/yamato/yaml/templating/#metadata